build.sh

```diff
@@ -56,6 +56,11 @@ mvn ${MAVEN_ARGS} dependency:copy-dependencies -DoutputDirectory="${BUILDDIR}"/a
 # Create the resource directory where all native libraries will be copied.
 mkdir -p "${BUILDDIR}"/all/
 
+source lib/VERSIONS
+VERSION="${VIPS_VERSION}-$(git rev-parse --short HEAD)"
+
+(cd script/enum-generator && python3 EnumGenerator.py "${VIPS_VERSION}")
```

Comment:
It will fail on Ubuntu. Can you please introduce a PYTHON_BIN variable like CMAKE_BIN?
`PYTHON_BIN=$(which python3|| which python)`